openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -358,12 +360,11 @@ def process_response(self, request, response):
         cookie.
 
         Also, the session cookie is deleted if prior verification failed
-        or the designated user in the request has changed since the
-        original request.
+        or the new cookie can't be created for some reason.
 
         Processing the response is a multi-step process, as follows:
 
-        Step 1. Call the parent's method to generate the basic cookie.
+        Step 1. Call the parent's method to (maybe) generate the basic cookie.
```

Comment:
Because, as we've learned, the response doesn't always set a cookie! But maybe it used to? Unclear!

Reply:
Is this change something we need to look into, or just a note with. I known consequence other than the change itself?

Reply:
I don't _think_. Honestly, the old behavior was kind of weird, if basically harmless.